src/ingestion/models/ingestionManager.ts

```diff
@@ -160,78 +172,93 @@ export class IngestionManager {
     await this.isValidRequestInputs(partData, inputFiles);
 
     //catalog ,mapproxy, jobmanager validation
-    await this.isInMapProxy(metadata.productId, metadata.productType, RequestType.NEW);
+    await this.validateLayerDoesntExistInMapProxy(metadata.productId, metadata.productType);
```

Comment:
check if you can pass here just the maproxyLayerId instead productId and productType

Reply:
why? the function creates the layer name inside. creating one line functions is not a good practice

Reply:
then remove the method "getlLayerExistanceInMapProxy" and use 
`const existanceInMapServer = await this.mapProxyClient.exists(layerName);`

```
const fullLayerName = getMapServingLayerName(productId, productType);
validateLayerExistsInMapProxy(fullLayerName);
```

Reply:
you asked for this implementation a week ago

Reply:
it is...
but i also asked to remove the double params..

Reply:
done